modules/lib/lib-project/src/test/resources/test/ListProjectsHandlerTest.js

```diff
@@ -0,0 +1,61 @@
+var assert = require('/lib/xp/testing.js');
+var project = require('/lib/xp/project.js');
+
+var project1Expected = {
+    name: 'myproject1',
+    displayName: 'project display name 1',
+    description: 'project description 1',
+    permissions: {
+        owner: [
+            'user:system:owner2',
+            'user:system:owner1'
+        ],
+        viewer: [
+            'user:system:viewer1'
+        ]
+    },
+    readAccess: {
+        public: false
+    }
+};
+
+var project2Expected = {
+    name: 'myproject2',
+    displayName: 'project display name 2',
+    description: 'project description 2',
+    permissions: {},
+    readAccess: {public: false}
+};
+
+exports.listProjects = function () {
+    assert.assertJsonEquals([], project.list());
+
+    createProject1();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['createProject1' was used before it was defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5511138)